src/annot/loc.rs

```diff
@@ -0,0 +1,143 @@
+// Copyright 2017 Nicholas Ingolia
+// Licensed under the MIT license (http://opensource.org/licenses/MIT)
+// This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Trait shared across sequence locations -- spliced, contiguous, or
+//! single-position.
+
+use std::ops::Neg;
+
+use annot::contig::Contig;
+use annot::pos::Pos;
+
+use strand::*;
+
+pub trait Loc {
```

Comment:
Can you add a doc comment to the trait itself?

Reply:
Certainly, just pushed this.